app/assets/javascripts/3dbio_viewer/src/webapp/components/model-upload/ModelUpload.tsx

```diff
@@ -65,6 +68,15 @@ export const ModelUpload: React.FC<ModelUploadProps> = React.memo(props => {
 
     const submit = useCallbackEffect(submitCb);
 
+    const downloadExample = React.useCallback<React.MouseEventHandler<HTMLAnchorElement>>(
+        ev => {
+            ev.stopPropagation();
+            ev.preventDefault();
+            compositionRoot.downloadAnnotationsExample.execute();
```

Comment:
[minor] Rule of thumb: return always the execute result, it may be a Cancel which can be then used in a cancellation.

Reply:
In this case is not Future or a Promise, should I create it?

Reply:
No, we can keep it `void`. But at some point we have a (cancellable) Promise/Future, the return will come in handy.